tests/test_cells_and_edges.py

```diff
@@ -289,11 +289,11 @@ def test_average_hexagon_area():
 
 def test_average_hexagon_edge_length():
     expected_in_km = {
-        0: 1107.712591000,
-        1: 418.676005500,
-        2: 158.244655800,
-        9: 0.174375668,
-        15: 0.000509713,
+        0: 1281.256011,
+        1: 483.0568391,
+        2: 182.5129565,
+        9: 0.200786148,
+        15: 0.000584169,
```

Comment:
This is a pretty big change. Do we know what happened here?

Reply:
This appears to be reflecting https://github.com/uber/h3/pull/749

Reply:
OK, I guess I thought we had this in a while ago.